boa3/model/operation/binary/logical/logicor.py

```diff
@@ -27,6 +27,10 @@ def validate_type(self, *types: IType) -> bool:
         left: IType = types[0]
         right: IType = types[1]
 
+        if left.is_type_of(int):
+            left = Type.int
+            right = Type.int
+
         return left == right and left in self._valid_types
```

Comment:
I think the problem is in the [`|` operation](https://github.com/CityOfZion/neo3-boa/blob/development/boa3/model/operation/binary/logical/logicor.py) and not in the type itself. 

The problem is that the compiler doesn't accept `FindOptions` type as an argument for the bitwise or because it only accepts `int` and `bool`. It's not recognizing that `FindOptions` values are `int` values

Maybe this is a better solution:
```suggestion
        return left == right and any(_type.is_type_of(left) for _type in self._valid_types)
```